cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/statements/LabelStatement.kt

```diff
@@ -48,6 +50,14 @@ class LabelStatement : Statement() {
             .toString()
     }
 
+    override var statementEdges: MutableList<PropertyEdge<Statement>>
+        get() =
+            subStatement?.let { mutableListOf(PropertyEdge(this, it, mutableMapOf())) }
+                ?: mutableListOf()
+        set(value) {
+            subStatement = value.firstOrNull()?.end
+        }
+
```

Comment:
I would prefer using the `wrap` / `unwrap` functions here for consistency.

```suggestion
    override var statementEdges: MutableList<PropertyEdge<Statement>>
        get() =
            subStatement?.let { mutableListOf(PropertyEdge(this, it, mutableMapOf())) }
                ?: mutableListOf()
        set(value) {
            subStatement = value.firstOrNull()?.end
        }

```